x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/scalar/multivalue/MvCount.java

```diff
@@ -37,18 +37,18 @@ public MvCount(
         @Param(
             name = "v",
             type = {
-                "unsigned_long",
-                "date",
                 "boolean",
+                "cartesian_point",
+                "date",
                 "double",
-                "ip",
-                "text",
+                "geo_point",
                 "integer",
+                "ip",
                 "keyword",
-                "version",
                 "long",
-                "geo_point",
-                "cartesian_point" }
+                "text",
+                "unsigned_long",
+                "version" }
```

Comment:
I wonder if we can write a checkstyle rule that forces these to be alphabetical!